cvat/apps/engine/urls.py

```diff
@@ -24,6 +24,7 @@
 router.register('cloudstorages', views.CloudStorageViewSet)
 router.register('assets', views.AssetsViewSet)
 router.register('guides', views.AnnotationGuidesViewSet)
+router.register('requests', views.RequestViewSet,  basename="request")
```

Comment:
I'm not sure that I understand which pattern you mean. Are you talking about using plural instead of singular? In that case, looks like the standard pattern is `ObjectViewSet`..

Reply:
Yes, I meant changing it to `RequestsViewSet`, but it seems it was my bad.